### PR TITLE
[CUBRIDQA-1214] Update xkill

### PR DESCRIPTION
**Pull request by tw-kang**
http://jira.cubrid.org/browse/CUBRIDQA-1214

### CODE REVIEW

File: CTP/shell/init_path/init.sh [hunk 1]
```diff
@@ -1071,7 +1071,7 @@ function xkill
    if [ "$OS" == Windows_NT ]
    then
        win_svr_pid=`get_win_service_pid`
```
Comment (tw-kang) [inline CTP/shell/init_path/init.sh]: need to upgrade white list.
need test for 'xkill cub' with -w option.

File: CTP/shell/init_path/init.sh [hunk 2]
```diff
@@ -1122,7 +1109,26 @@ function xkill_java_windows {
...
+        Name = 'wininit.exe' or 
+        Name = 'winlogon.exe' or 
+        Name = 'services.exe' or 
+        Name = 'lsass.exe' or 
+        Name = 'svchost.exe' or 
+        Name = 'conhost.exe' or 
+        Name = 'WmiPrvSE.exe' or
+        Name = 'taskhostex.exe' or 
+        Name = 'dwm.exe' or 
+        Name = 'rdpclip.exe'
```
Comment (swi0110) [inline CTP/shell/init_path/init.sh]: (in my opinion)
How about adding to 'explorer.exe'??
That is the Windows OS default process.
if it is destroyed then auto-restart the process or the OS does strange.

Comment (tw-kang) [reply CTP/shell/init_path/init.sh]: killing explorer.exe will end the taskbar and desktop environment, significantly impacting the user's ability to interact with the system.
 the review is reflected.

**Review by swi0110 — APPROVED** [review]

File: CTP/shell/init_path/init.sh (hunk 1 above)
Comment (tw-kang) [reply CTP/shell/init_path/init.sh]: reinforcing white list is needed.

**Review by ssihil — APPROVED** [review]

**Review by kwonhoil — APPROVED** [review]

**Pull request merged**